docs/building-on-etherlink/bridging-xtz.md

```diff
@@ -15,19 +17,17 @@ These bridges are trustless and permissionless; anyone can use them without rest
 - Bridging tokens from Tezos layer 1 to Etherlink is referred to as _depositing_ tokens.
 - Bridging tokens from Etherlink to Tezos layer 1 is referred to as _withdrawing_ tokens.
 
-:::note Bridging time
-Tokens that you bridge from Tezos layer 1 to Etherlink are available for use on Etherlink immediately.
-
-Tokens that you bridge from Etherlink to Tezos layer 1 are available for use on Tezos in two weeks.
+<WarningBox 
```

Comment:
It's possible to override the default `:::note` box to substitute a custom MDX component. That would make it easier because the text could stay in markdown instead of being in a quoted string in the custom component. This could be a follow-up because we want to get this out soon.

Reply:
I will fix this in this PR